libraries/chain/kv_context_rocksdb.cpp

```diff
@@ -213,16 +217,17 @@ namespace eosio { namespace chain {
                old_value = view.get(contract, { key, key_size });
                if (!old_value)
                   return 0;
+               EOS_ASSERT(old_value->size() >= kv_payer_size, kv_rocksdb_bad_value_size_exception , "The size of value returned from RocksDB is less than payer's size");
```

Comment:
initialize a parameter actual_size before try-catch, set it here and use to pass to erase_table_usage

Reply:
OK